lib/exavier/config.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Exavier.Config do
+  @config_file ".exavier.exs"
+  @config_file_content File.read!("priv/" <> @config_file)
+
+  def get(key, default \\ nil) do
+    file = config_file()
+
+    unless exists?() do
+      raise("Config file #{file} not found")
+    end
+
+    {configs, _binding} =
+      file
+      |> File.read!()
+      |> Code.eval_string()
+
+    Map.get(configs, key, default)
+  end
+
+  def exists?, do: config_file() |> File.exists?()
+
+  def config_file, do: @config_file
```

Comment:
You'll probably want to do a recursive search for this file in any directory from the application root, maybe using something like: https://hexdocs.pm/elixir/Application.html#app_dir/1 and https://hexdocs.pm/elixir/Path.html#wildcard/2 to get there?

Reply:
I'd prefer we keep it as a _"root directory file"_ (relative to the project path) for now (i.e., `./.exavier.exs`). Random recursive search might be a needless pain to have here, unless you have a strong argument in favor of it?

Reply:
I wouldn’t say it’s a strong argument for this specifically, but you should consider how you want to handle alternative project structures (like umbrella projects, for example).

It’s also totally cool to say you don’t support those if you want!

Reply:
I think it's fair for people right now to have to `cd` into the directory they want to run `exavier` in. If umbrella projects become an increasing concern, we can revisit. Sounds fair? I'll merge as is now.

Reply:
I have an umbrella app that is configured in the way it's not possible to just `cd` into a directory of an individual test and run it, so `exavier` simple don't work here.
Just my `+1` to umbrella support.